dist/purecounter_vanilla.js

```diff
@@ -1 +1,2 @@
+//# sourceMappingURL=purecounter_vanilla.js.map
```

Comment:
in "hide whitespaces changes" mode you can see that this file did not change, only the source map was added